packages/core/src/index.ts

```diff
@@ -35,27 +32,6 @@ export * from './types'
 
 const __EMOTION_VERSION__ = packageInfo.version
 
-const getCSS = (props) => {
-  if (!props.sx && !props.css) return undefined
-  return (theme) => {
-    const styles = css(props.sx)(theme)
-    const raw = typeof props.css === 'function' ? props.css(theme) : props.css
-    return [styles, raw]
-  }
-}
-
-const parseProps = (props) => {
-  if (!props) return null
-  const next: typeof props & { css?: InterpolationWithTheme<any> } = {}
-  for (let key in props) {
-    if (key === 'sx') continue
-    next[key] = props[key]
-  }
-  const css = getCSS(props)
-  if (css) next.css = css
-  return next
-}
-
 export const jsx: typeof React.createElement = (type, props, ...children) =>
```

Comment:
Random idea: If the automatic runtime is the new default, then we could move this `jsx` function to a separate package to decrease bundle size.

This would be a breaking change, so we could do it gradually by creating the package, importing the function here and display a little deprecation warning during development that this will be removed in the next release or `v1.0.0`.

Thoughts on this?

Reply:
I noticed recently how they do it in CRA. Maybe it helps. In create react app the new jsx runtime is auto-enabled or -disabled by this code at https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js#L74
```

const hasJsxRuntime = (() => {
  if (process.env.DISABLE_NEW_JSX_TRANSFORM === 'true') {
    return false;
  }

  try {
    require.resolve('react/jsx-runtime');
    return true;
  } catch (e) {
    return false;
  }
})();

```

Reply:
Thanks for the info @RobIsHere! I think we can't use this approach because then we would assume that everyone with React v16.14.0 or newer uses the new runtime, although not everyone uses it. CRA can enforce the automatic runtime whenever it's available because CRA itself controls the babel config of a project and can modify the config there respectively. But we can't do that in Theme UI, we need to support both ways for now.

Reply:
`jsx` function is a one-liner, we'll just keep it, doesn't make sense to move anywhere.